packages/eve-runtime/src/github/operator.ts

```
@@ -0,0 +1,301 @@
+        installationId: input.installationId,
+        method: "POST",
+        path: `/repos/${pathPart(input.owner)}/${pathPart(input.repo)}/issues/${request.targetNumber}/labels`,
+      });
+      return { resourceId: String(request.targetNumber) };
+    case "rerun_failed_workflow":
+      await verifyIssue(input, request.issueNumber);
+      const workflow = await githubRequest<{ run_attempt?: number }>({
+        installationId: input.installationId,
+        method: "GET",
+        path: `/repos/${pathPart(input.owner)}/${pathPart(input.repo)}/actions/runs/${request.workflowRunId}`,
+      });
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)